quic/s2n-quic-transport/src/connection/mod.rs

```diff
@@ -81,4 +81,6 @@ pub struct Parameters<'a, Cfg: endpoint::Config> {
     pub supervisor_context: &'a supervisor::Context<'a>,
     /// The event subscriber for the endpoint
     pub event_subscriber: &'a mut Cfg::EventSubscriber,
+    // The datagram provider for the endpoint
+    pub datagram_endpoint: &'a mut Cfg::DatagramEndpoint,
```

Comment:
nit: i generally try and not leave comments like these but its going to bother me going forward so i am making an exception.

Can we keep event_subscriber last. events is meta to the library and least important param to the quic protocol functionality. When reading code its nice to be able to filter out the unimportant parameters.

```suggestion
    pub datagram_endpoint: &'a mut Cfg::DatagramEndpoint,
    pub event_subscriber: &'a mut Cfg::EventSubscriber,
```

Reply:
Yeah that's a good point, feel free to point this out anywhere else in the PR. I'm not really sure what is important or not still.